### PR TITLE
Make ZenDesk use v2

### PR DESCRIPTION
**Pull request by LucGenetier**
ZD CDP connector now implements version 2

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by jas-valgotar — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs [hunk 1]
```diff
@@ -90,7 +90,10 @@ public async Task<ConnectorType> ResolveTableAsync(string tableName, Cancellatio
             return ConnectorFunction.GetCdpTableType(this, connectorName, "Schema/Items", FormulaValue.New(text), sqlRelationships, ConnectorCompatibility.CdpCompatibility, _tabularTable.DatasetName, out string name, out string displayName, out TableDelegationInfo delegationInfo);
         }
 
-        private bool IsSql() => _uriPrefix.Contains("/sql/");
+        internal static bool IsSql(string uriPrefix) => uriPrefix.Contains("/sql/");
+
+        internal static bool UseV2(string uriPrefix) => uriPrefix.Contains("/sql/") ||
+                                                        uriPrefix.Contains("/zendesk/");
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs:96]: > uriPrefix.Contains("/zendesk/"); [](http://example.com/codeflow?start=55&length=33)

Does Power Apps also have this same check?

**Review by MikeStall — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**